spec/actions_spec.rb

```diff
@@ -136,18 +136,22 @@ def self.name
       non_existing_id = 1
 
       expect {
-        klass.find(non_existing_id)
+        begin
+          klass.find(non_existing_id)
+        rescue Redis::CommandError => e
```

Comment:
I suspect this has something to do with redis's cluster implementation. somehow it thinks the cluster is down when failed to find a record on CI otherwise it causes flakiness.

Will add a comment to explain it